synapse/axon.py

```diff
@@ -1574,6 +1580,47 @@ async def wput(self, sha256, url, params=None, headers=None, method='PUT', ssl=T
                     'mesg': mesg,
                 }
 
+    async def _flatten_clientresponse(self,
+                                      resp: aiohttp.ClientResponse,
+                                      toplevel=True,
+                                      ) -> dict:
+        info = {
+            'ok': True,
+            'url': str(resp.url),
+            'code': resp.status,
+            'headers': dict(resp.headers),
+            'request': {
+                'url': str(resp.request_info.url),
+                'headers': dict(resp.request_info.headers),
+                'method': str(resp.request_info.method),
+            }
+        }
+
+        hashset = s_hashset.HashSet()
```

Comment:
could move this into the toplevel block since it only gets used there

Reply:
moved